README.md

```diff
@@ -48,78 +48,17 @@ pod 'JSONCodable', '~> 2.1'
 
 ---
 
-`JSONCodable` is made of two separate protocols `JSONEncodable` and `JSONDecodable`.
-`JSONEncodable` allows your structs and classes to generate `NSDictionary` or `[String: AnyObject]` equivalents for use with `NSJSONSerialization`.
-`JSONDecodable` allows you to generate structs from `NSDictionary` coming in from a network request for example.
+**JSONCodable** is made of two separate protocols `JSONEncodable` and `JSONDecodable`.
 
-We'll use the following models in this example:
-```swift
-struct User {
-    let id: Int
-    let name: String
-    var email: String?
-    var company: Company?
-    var friends: [User] = []
-}
-
-struct Company {
-    let name: String
-    var address: String?
-}
-```
```

Comment:
I think it's confusing to look an extension for struct without this context. We can split this up and put it next to the extension - maybe that might be clearer?